templates/workspace_services/guacamole/guacamole-server/guacamole-auth-azure/src/main/java/org/apache/guacamole/auth/azuretre/connection/ConnectionService.java

```diff
@@ -62,7 +62,7 @@ public static Map<String, Connection> getConnections(final AzureTREAuthenticated
         return connections;
     }
 
-    private static Map<String, GuacamoleConfiguration> getConfigurations(final AzureTREAuthenticatedUser user)
+    public static Map<String, GuacamoleConfiguration> getConfigurations(final AzureTREAuthenticatedUser user)
```

Comment:
Why public?

Reply:
so you can call it from the provider before you create the conntext

Reply:
But getConfigurations is only called inside the class

Reply:
left overs. can revert